dotcom-rendering/src/model/enhanceCollections.ts

```diff
@@ -44,6 +44,12 @@ const enhanceCards = (collections: FEFrontCard[]): DCRFrontCard[] =>
 			};
 		});
 
+const decideContainerPalette = (
+	metadata: { type: FEContainerPalette }[],
+): DCRContainerPalette => {
+	return metadata[0].type;
```

Comment:
Would the idea be that we can return certain style properties from this function?

Reply:
The idea is that we can take the array of tags (where there are currently conventions where you sometimes have two tags) and rationalise it into one value by putting all the magic incantations here, in one place